src/pages/DepositSwiftPage/index.tsx

```diff
@@ -0,0 +1,253 @@
+                    {this.renderField('accountNumber', 'Account number')}
+                    {this.renderField('accountName', 'Account name')}
+                    {this.renderField('bankAddress', 'Bank address')}
+                    {this.renderField('companyAddress', 'Company Address')}
+                    {this.renderField('purposeOfPayment', 'Purpose of Payment')}
+                    {this.renderField('bic', 'BIC')}
+                    {this.renderField(
+                      'correspondentAccount',
+                      'Correspondent Account'
+                    )}
+                  </div>
+
+                  <div className="deposit-swift-form__links">
+                    <a
+                      className="link"
+                      href="https://www.lykke.com/terms_of_use"
+                      target="_blank"
+                    >
+                      Terms of Use
+                    </a>
+                  </div>
+
+                  <div className="deposit-swift-form__actions">
+                    <input
+                      type="submit"
+                      value="Send to email"
+                      className="btn btn--primary"
+                      disabled={formikBag.isSubmitting || !formikBag.isValid}
+                    />
+                    <a
+                      href="#"
+                      onClick={this.props.history.goBack}
+                      className="btn btn--flat"
+                    >
+                      Cancel and go back
+                    </a>
+                  </div>
+                </Form>
+              )}
+            />
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  private renderField = (name: string, label: string) => (
+    <div className="row">
+      <div className="col-sm-12">
+        <Field
+          name={name}
+          // tslint:disable-next-line:jsx-no-lambda
+          render={({field, form}: FieldProps<DepositSwiftModel>) => (
+            <div className="deposit-swift-form__form-group">
+              <div>
+                <label className="control-label">{label}</label>
+                <div className="value">{field.value || 'Not available'}</div>
+              </div>
+              <div>
+                <CopyToClipboard
+                  text={field.value}
+                  onCopy={text => this.handleCopy(text, form.setStatus)}
+                >
+                  <button className="btn btn--icon" type="button">
+                    <i className="icon icon--copy_thin" />
+                  </button>
+                </CopyToClipboard>
+                {field.value &&
+                  form.status === field.value && (
+                    <small className="copy-to-clipboard-message">Copied!</small>
+                  )}
+              </div>
+            </div>
+          )}
+        />
+      </div>
+    </div>
+  );
+
+  private readonly handleCopy = (
+    text: string,
+    setStatus: (status?: any) => void
```

Comment:
Do we have type for status? Or it can be anything?

Reply:
Can be anything https://github.com/jaredpalmer/formik#setstatus-status-any--void